### ISSUE TITLE
Question about  google_api_repository.py

### ISSUE THREAD

**Issue by QuchenFu** [brunnurs/valuenet#2]
Hello,

I am trying to do manual inference on the pretrained_model, however, I can not find the google_api_repository.py file. Can you elaborate on how I can set the API in your code? Thank you!

Best

**Comment by QuchenFu** [comment]
valuenet/src/named_entity_recognition/api_ner/google_api_repository.py

**Comment by brunnurs** [comment]
@QuchenFu try which metric are you referring to with 45%? The official Spider metric is the one I report with "all" (so all difficulty levels). 
In general I don't think the batch size will have such a large impact, as I often also trained with a batch size of 8 or 16. See the screenshots attached for exact configuration of one of my best run.
![Screenshot from 2020-06-22 09-52-15](https://user-images.githubusercontent.com/1187124/85262498-234ef200-b46e-11ea-9177-f5cdf0b5482d.png)
![Screenshot from 2020-06-22 09-52-06](https://user-images.githubusercontent.com/1187124/85262500-23e78880-b46e-11ea-8e72-0d8362e2051f.png)

**Comment by QuchenFu** [comment]
Thank you so much for clarify this! I thought "accuracy" in the plot refers to the accuracy of all levels. If "all" is the "all difficulty level", what does the "accuracy" metric refer to(down in the middle of the plot)? @brunnurs

**Comment by brunnurs** [comment]
"accuracy" is referring to the accuracy on SemQL level. We compare the predicted SemQL with the expected SemQL ground truth. Only afterwards we transform the predicted SemQL to sql and execute it with the spider evaluation scripts. 
The difference between "accuracy" and "all" stems from the fact that the SemQL-to-SQL transformation is not perfect, but then sometimes a query also works if the SemQL is not exactly a 1:1 match to the ground truth.

For more information about the "accuracy" metric have a look at the code starting here: https://github.com/brunnurs/valuenet/blob/28c8a65d0359147bd21415026fe5392170fd3a57/src/evaluation.py#L50
